### PR TITLE
fix(user-feedback): Fix user feedback page hostname

### LINKED ISSUE #13970: endpoint in error-page-embed.js does not respect system.url-prefix
## Important Details

How are you running Sentry?

* [x ] On-Premise docker [9.1.1]
## Description

using raven-js with angular works as expected to capture exceptions.
when reporting errors we get the wrong endpoint from the sentry instance

javascript snippet:
```js
Raven
  .config(`https://${environment.sentryKey}@MY_INSTANCE/PROJECT_NUMBER`,
    {
      environment: environment.production ? 'prod' : 'dev',
      release: versions.version
    }
  )
  .install();

export class RavenErrorHandler implements ErrorHandler {
  handleError(error: any): void {
    const err = error.originalError || error;
    console.error(err);
    Raven.captureException(err);
    if (true) {
      // fetches error-page-embed.js from sentry instance with wrong endpoint (docker endpoint)
      Raven.showReportDialog();
    }
  }
}
```

this end point is set here: https://github.com/getsentry/sentry/blob/master/src/sentry/web/frontend/error_page_embed.py
```python
# using request.build_absolute_uri() instead of the value of system.url-prefix 'endpoint':  
 
mark_safe('*/' + json.dumps(request.build_absolute_uri()) + ';/*'),
```

## Steps to Reproduce

web:9000 is the hostname of the docker container
screenshot3

### What you expected to happen

system.url-prefix in config.yaml: error-page-embed.js should be respected

cc/ @max-wittig

### PR DESCRIPTION
**Pull request by BYK**
_Courtesy of @max-wittig._

Fixes #13970

### CODE REVIEW

**Comment by max-wittig** [comment]
@BYK thanks for creating the MR! :) I think I wanted to test before submitting first, but forgot about it.

**Comment by max-wittig** [comment]
@gocarlos

File: src/sentry/web/frontend/error_page_embed.py [hunk 1]
```diff
@@ -208,7 +209,7 @@ def dispatch(self, request):
         )
 
         context = {
-            "endpoint": mark_safe("*/" + json.dumps(request.build_absolute_uri()) + ";/*"),
+            "endpoint": settings.SENTRY_OPTIONS["system.url-prefix"],
```
Comment (markstory) [inline src/sentry/web/frontend/error_page_embed.py]: I think you need the surrounding `*/` and `;/*`. The template uses this variable in the following way:

```
  var endpoint = /*{{ endpoint }}*/'';
```
Which won't work with the current value.

Comment (BYK) [reply src/sentry/web/frontend/error_page_embed.py]: Good now?

PS: I have no idea about how to test this.

Comment (markstory) [reply src/sentry/web/frontend/error_page_embed.py]: This endpoint serves the user-feedback dialog. You can test it by setting up the browser-sdk https://docs.sentry.io/enriching-error-data/user-feedback/?platform=browser

**Comment by BYK** [comment]
@markstory alright, tested locally and verified that this works. The dialog errored due to a CORS error but I'm guessing that's a server-side config issue or a bug.

**Review by markstory — APPROVED** [review]

**Pull request merged**